corehq/apps/user_importer/importer.py

```diff
@@ -658,11 +661,17 @@ def _process_web_user(self):
                     if cv["profile_name"]:
                         _check_profile(cv["profile_name"], self.domain_info.profiles_by_name)
                         profile = self.domain_info.profiles_by_name[cv["profile_name"]]
+                    tableau_group_ids = None
+                    tableau_role = cv["tableau_role"] if "tableau_role" in cv else None
```

Comment:
I think similar to what you did for `WebUserRow`, you can parse for `tableau_role` within `_process_column_values`